ui/components/app/import-token/token-list/token-list.component.js

```diff
@@ -14,60 +30,104 @@ export default class TokenList extends Component {
     results: PropTypes.array,
     selectedTokens: PropTypes.object,
     onToggleToken: PropTypes.func,
+    currentNetwork: PropTypes.object,
+    testNetworkBackgroundColor: PropTypes.object,
   };
 
   render() {
     const {
       results = [],
       selectedTokens = {},
+
       onToggleToken,
       tokens = [],
+      currentNetwork,
+      testNetworkBackgroundColor,
     } = this.props;
 
     return results.length === 0 ? (
-      <TokenListPlaceholder />
+      <Box paddingLeft={4} paddinfRight={4} className="emptyListTokens">
```

Comment:
The `className` here is too generic; we follow the BEM pattern in our codebase, so maybe:

`token-list__empty-list`